jenkins-scripts/dsl/ignition.dsl

```diff
@@ -553,6 +553,29 @@ ignition_software.each { ign_sw ->
         }
     }
   }
+
+  // 3. install jobs to test bottles
+  supported_install_pkg_branches(ign_sw).each { major_version, supported_distros ->
+    def install_default_job = job("ignition_${ign_sw}${major_version}-install-homebrew-bottle")
```

Comment:
Although we don't have a clear policy on job naming, my intention has been to reserve the last post-fixes to indicate platform. We can include the bottle information together with the install word, something like: 

```suggestion
    def install_default_job = job("ignition_${ign_sw}${major_version}-install_bottle-homebrew-amd64")    
```

Reply:
regenerated bottle jobs with new names in https://build.osrfoundation.org/job/_dsl_ignition/977/